### ISSUE TITLE
Behavior of keyboard capture/element focus does not match EVE

### ISSUE THREAD

**Issue by ghost** [evevision/evevision#11]
When clicking inside of the Welcome splash screen or the Tools screen, it draws keyboard focus away from the client.  You're unable to press shortcuts like F1 or Ctrl-Space when those windows have focus, even though there are no text fields that they could be awaiting input for.

As mentioned on Discord, for other active windows (Dotlan, etc.)there really isn't a way around this because it may have a text field and will need to have keyboard focus there to receive the input.

**Comment by evevision** [comment]
After some testing, I have observed the following:

- EVE does not 'capture' the keyboard anywhere except in text inputs. You can use all your keyboard shortcuts, including single-character ones, until you've clicked into a text input.
- Once you've focused on a text input, EVE will:
  - capture any characters, i.e. hotkeys bound to just 'A' will not work
  - capture any editing keys, i.e. arrow keys, page up/down, home
  - pass through ctrl/alt+anything combination hotkeys (i.e. ctrl+s will work, but shift+a will be captured)
  - pass through any hotkey of non character/editor keys - i.e. f1-f12
- EVE does not clear focus by clicking on the background of windows

For our own windows where we can know if we're in an input, we should be able to match EVE's behavior perfectly.

External tools (ChildWindows) may be a bit more difficult. I need to inject JS into them to determine if you are focused on a text input, and then somehow get that state back to the main process from within the external sandbox.

**Comment by evevision** [comment]
Another issue to consider is external tools or websites that would actually want to capture those controls like F1-F12. Perhaps such websites shouldn't be in EveVision in general since they would break UI parity. But, if we did want to add them, it would be trivial to change the input handling for them on a case-by-case basis.

**Comment by evevision** [comment]
One more thing - navigational keys (arrow keys, page up/down, home, end) should always work if your mouse is over the window, and not because you are focused on an EveVision window or captured in a text input. They are necessary for many parts of web navigation.

**Comment by evevision** [comment]
In the case of the generic webbrowser (not statically added external tools), I believe it should have some toggle to allow fully capturing your input when the window is focused. There's no reason to prevent that.
